jalien-xrootd.sh

```diff
@@ -0,0 +1,59 @@
+package: JAlien-XRootD
+version: "%(tag_basename)s"
+tag: 4.8.3
+build_requires:
+ - CMake
+ - "OpenSSL:(?!osx)"
+ - "osx-system-openssl:(osx.*)"
+ - ApMon-CPP
+ - libxml2
+ - "GCC-Toolchain:(?!osx)"
+---
+#! /bin/bash -e
+
+PACKED_NAME="xrootd-$GIT_TAG.tar.gz"
```

Comment:
We build XRootD from Git sources, we don't unpack tarballs. The existing XRootD recipe has proper references already.